.github/workflows/all.yml

```diff
@@ -305,3 +305,18 @@ jobs:
         run: |
           mv tools/semgrep/.semgrepignore . & \
           semgrep --verbose --config="r2c" --config="tools/semgrep/rules/example.yaml"
+
+  lint-cargo-toml:
+    name: lint-cargo-toml
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install dprint
+        run: sudo snap install dprint
```

Comment:
We can't install dprint with `snap` as it fails in CI, logs here https://github.com/build-trust/ockam/actions/runs/4708494194/jobs/8350973001?pr=4723#step:3:5. Can we instead use npm `npm install -g dprint`?